training/tests/integration/aicon/test_cicd_aicon_04_icon-dream_medium.yaml

```diff
@@ -171,10 +171,8 @@ model:
     _target_: anemoi.models.models.encoder_processor_decoder.AnemoiModelEncProcDec
 
   num_channels: 5 # reduced for CI/CD test. original value 512
-  activation: GELU
```

Comment:
Where is the activation now specified for this test?

Reply:
In the layer kernels

Reply:
If DWD has the layer kernel explicitly defined as GELU for their tests, wouldn't it be better to state clearly the layer kernel section with that activation as part of this config? (we can ping Marek if useful)

Reply:
I have added the layer_kernels GELU definition here. cc: @MeraX